scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/example/nnframes/xgboost/xgbClassifierTrainingExampleOnCriteoClickLogsDataset.scala

```diff
@@ -108,6 +125,9 @@ object xgbClassifierTrainingExampleOnCriteoClickLogsDataset {
     // randomly split dataset to (train, eval1, eval2, test) in proportion 6:2:1:1
     val Array(train, eval1, eval2, test) = xgbInput.randomSplit(Array(0.6, 0.2, 0.1, 0.1))
```

Comment:
you can cache these train eval test, and count the time of below code, this time is data preprocess time.
```
train.cache()
eval1.cache()
eval2.cache()
test.cache()
train.count()
```